easybuild/easyconfigs/s/SAMtools/SAMtools-0.1.19-intel-2015a.eb

```diff
@@ -0,0 +1,21 @@
+name = 'SAMtools'
+version = '0.1.19'
```

Comment:
yes, but it's incompatible with the 0.x version, I can add the 1.1 easyconfig here if you like, but It's not required for cufflinks and I didn't wat to put to much in one pull request ;-)